spec/support/acts_as_tenant.rb

```diff
@@ -9,10 +9,12 @@ def with_tenant(tenant, &proc)
     context "with tenant #{tenant.inspect}" do |*params|
       before(:each) do
         ActsAsTenant.current_tenant = send(tenant)
+        @old_host = Capybara.app_host
```

Comment:
It worked fine before by setting the `request['host']` parameter? That only affected one instance.